rdt/transformers/pii/anonymizer.py

```diff
@@ -81,7 +81,7 @@ def _check_locales(self):
         missed_locales = []
         for locale in locales:
             spec = importlib.util.find_spec(f'faker.providers.{self.provider_name}.{locale}')
-            if spec is None:
+            if spec is None and not self.provider_name.endswith(locale):
```

Comment:
Is it safe to assume that if the provider name ends with the locale, then it exists? Is it possible it's just a bad provider name that ends with a locale

Reply:
I think that you are right, we can improve this by adding `.`. to the locale when looking for.

Reply:
I meant more that I think we should make sure the provider exists still. Like if the locale is already present, extract it and runs it through `importlib.util.find_spec(f'faker.providers....`